site/src/pages/WorkspacePage/ResourceMetadata.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import { MockWorkspaceResource } from "testHelpers/entities";
+import type { Meta, StoryObj } from "@storybook/react";
+import { ResourceMetadata } from "./ResourceMetadata";
+
+const meta: Meta<typeof ResourceMetadata> = {
+  title: "pages/WorkspacePage/ResourceMetadata",
+  component: ResourceMetadata,
+};
+
+export default meta;
+type Story = StoryObj<typeof ResourceMetadata>;
+
+export const Markdown: Story = {
+  args: {
+    resource: {
+      ...MockWorkspaceResource,
+      metadata: [
+        { key: "text", value: "hello", sensitive: false },
+        { key: "link", value: "[hello](#)", sensitive: false },
+        { key: "b/i", value: "_hello_, **friend**!", sensitive: false },
+        { key: "coder", value: "`beep boop`", sensitive: false },
```

Comment:
Should we have a `sensitive: true` test case?

Reply:
Yes, we should. I missed this comment so, I'm going to add it in a next PR.

Reply:
https://github.com/coder/coder/pull/11755